utils/servo/ServoType.java

```diff
@@ -1,16 +1,10 @@
 package frc.team670.mustanglib.utils.servo;
 
-/**
- * Represents a type of servo.
- * Each type has a {@link PWMConfig} attached to it, which should be used when setting PWM bounds.
- * 
- * @author AkshatAdsule
- */
 public enum ServoType {
     /**
      * Andymark L16 Linear Servo
      */
-    ANDYMARK_L16(new PWMConfig(2.0, 1.8, 1.5, 1.2, 1.0));
+    ANDYMARK_L16(new PWMConfig(2, 2, 1, 1, 1)); //not correct please change
```

Comment:
This was the 2023 and older PWM API to which these constants are passed: https://robotpy.readthedocs.io/projects/wpilib/en/latest/wpilib/PWM.html#wpilib.PWM.setBounds - so everything was in milliseconds
This is the 2024 API: https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/PWM.html#setBoundsMicroseconds(int,int,int,int,int) - so now everything is in microseconds

So you just need to scale the old numbers from milliseconds to microseconds and then this should be ok